tests/bin/install-wc-tests.sh

```diff
@@ -10,20 +10,15 @@ DB_USER=$2
 DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
-WC_VERSION=${6-"5.0.0"}
+WC_VERSION=${6-"7.4.1"}
 
 install_wc() {
-	git clone --depth=1 --branch=$WC_VERSION https://github.com/woocommerce/woocommerce.git /tmp/woocommerce
-	composer install -d /tmp/woocommerce
-
-	echo "Done unzipping WooCommerce\n"
+	git clone --depth=1 --branch=$WC_VERSION https://github.com/woocommerce/woocommerce.git /tmp/woocommerce && cd /tmp/woocommerce && nvm use && pnpm install && pnpm run build && echo "Done unzipping WooCommerce"
 }
 
 install_wp() {
-	echo "Installing WordPress via WooCommerce install script\n"
-	bash /tmp/woocommerce/tests/bin/install.sh $DB_NAME $DB_USER "$DB_PASS" $DB_HOST $WP_VERSION
-	echo "Done unzipping WordPress\n"
+	echo "Installing WordPress via WooCommerce install script" && bash /tmp/woocommerce/plugins/woocommerce/tests/bin/install.sh $DB_NAME $DB_USER "$DB_PASS" $DB_HOST $WP_VERSION && echo "Done unzipping WordPress"
 }
 
-install_wc
-install_wp
+rm -rf /tmp/woocommerce && install_wc && install_wp
```

Comment:
Added removing `/tmp/woocommerce` when running this script because otherwise your old WC would linger in `/tmp`.